website/src/css/custom.css

```diff
@@ -2143,4 +2143,66 @@ table th {
 [data-theme='dark'] table th {
   color: #000000; /* Black text on darker background */
   font-weight: bold;
-}
\ No newline at end of file
+}
+
+.tabs {
+  background-color: transparent;
+  padding: 10px;
+}
+
+/* Default color and border for tabs */
+.tabs__item {
+  background-color: #f9fbfc; /* Sets the tab color the same as the file borders */
+  border: 2px solid #ccc;
```

Comment:
Same note about using available color variables here, can we swap all uses of `#ccc` with a similar color currently available as a variable?